### PR TITLE
Yzrobot01

### PR DESCRIPTION
**Pull request by wyj4766**
add a new ROS robot platform from YZrobot

### CODE REVIEW

**Comment by wyj4766** [comment]
add a new ROS robot platform from YZrobot

File: _posts/2020-03-17-yzrobot-01b.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- '50kg payload'
+- 'auto charging dock'
+- 'ground'
+- 'indoor'
+- 'kinetic'
+- 'mobile robot'
+- 'yzrobot'
+twitter_text: Checkout the YZrobot-01B
+website: http://www.yzrobot.cn/en/ros-robot.html
+wiki_homepage: http://wiki.ros.org/Robots/yzrobot01b
```
Comment (tfoote) [inline _posts/2020-03-17-yzrobot-01b.md]: This page does not appear to exist. And is required following our contributing guide: https://robots.ros.org/contributing/

Comment (wyj4766) [reply _posts/2020-03-17-yzrobot-01b.md]: Dear: 
I modified the wrong wiki_homepage, please check again.
Thanks a lot!

**Comment by wyj4766** [comment]
modified wiki_homepage link

File: assets/img/robots/yzrobot-01b/readme [hunk 2]
```diff
@@ -0,0 +1 @@
+add a new folder
```
Comment (tfoote) [inline assets/img/robots/yzrobot-01b/readme]: This file is unnecessary.

Comment (wyj4766) [reply assets/img/robots/yzrobot-01b/readme]: Dear:
Sorry, now I deleted this unnecessary file
Thank you.

**Pull request merged**